### DISCUSSION TITLE
[Build Error]: Gatsby V5 build error - Built Rendering Engines failed validation failed validation -

### DISCUSSION THREAD

**Discussion by sukantatoptal** [gatsbyjs/gatsby#37552] [Help]
Recently upgraded to Gatsby V5 from v4. Started getting build error after the update. 

<img width="640" alt="image" src="https://user-images.githubusercontent.com/96305469/215675177-1003b2b2-6bd1-422c-914c-62eec71db0cf.png">

<img width="520" alt="image" src="https://user-images.githubusercontent.com/96305469/215675320-a7fd8fa2-77f0-4ecb-911b-d34e218b818a.png">

**Comment by Keshavdulal** [comment]
1. Did you try the default escape hatch?

```
# npm-escape-hatch.sh
rm -rf node_modules package-lock.json
gatsby clean
npm cache clean --force

npm install
```

```
# yarn-escape-hatch.sh
rm -rf node_modules yarn.lock
gatsby clean
yarn cache clean

yarn
```

2. Also, watch out for warning messages as such and upgrade them as necessary.
```
warn Plugin gatsby-plugin-emotion is not compatible with your gatsby version 5.5.0 - It requires gatsby@^4.0.0-next
```

3. Try sharing your entire build log and package.json for reference
